components/expenses/ConfirmProcessExpenseModal.tsx

```diff
@@ -14,7 +15,7 @@ import { toast } from '../ui/useToast';
 
 const messages = defineMessages({
   reasonPlaceholder: {
-    defaultMessage: 'e.g. Email Address is wrong',
+    defaultMessage: 'e.g. Same expense was submitted twice.',
```

Comment:
We need a stronger reason here

```suggestion
    defaultMessage: "e.g., We never worked with this person.",
```